docs/usage.md

```diff
@@ -122,6 +134,8 @@ To further assist in reproducbility, you can use share and re-use [parameter fil
 If you wish to share such profile (such as upload as supplementary material for academic publications), make sure to NOT include cluster specific paths to files, nor institutional specific profiles.
 :::
 
+If the pipeline input contains microbiomes of type `taxa` it may not generate the same results, as for each taxid the largest assembly is chosen, wihch might change. Therefore, using a specific assembly_id as explained in the section `Input type taxa` is highly recommended.
```

Comment:
```suggestion
If the pipeline input contains microbiomes of type `taxa` it may not generate the same results, as for each taxid the largest assembly is chosen, which might change. Therefore, using a specific assembly_id as explained in the section `Input type taxa` is highly recommended.
```
Same as above, describe setting to reproduce results, to make clear that in the firt run the pipeline can be used to obtain assembly_ids, if not given.